Builds/XCode/README.md

```diff
@@ -164,6 +169,12 @@ scons
 
 See: [here](https://ripple.com/wiki/Rippled_build_instructions#Building)
 
+## Unit Tests (Recommended)
+
+rippled builds a set of unittest into the server executable. To run these unit
+tests after building, pass the `--unittests` option to the compiled `rippled`
```

Comment:
I'm pretty sure the command line argument should be `--unittest` without the pluralizing "s".